src/index.js

```diff
@@ -30,7 +30,7 @@ const reduceMatchedKeyPaths = (obj, keyPath) => {
 }
 
 const loader = function (content) {
-  if (typeof this.query === 'string') {
+  if (typeof this.query !== 'string') {
```

Comment:
this is intended the user to use webpack 2 `query` object in webpack config, this package doesn't support string currently